### PR TITLE
[release] adding 7.25.0-rc.1 and 6.25.0-rc.1 entries.

### PR DESCRIPTION
**Pull request by truthbk**
### What does this PR do?

Adding 7.25.0~rc.1 and 6.25.0~rc.1 release entries.

### Motivation

Prep-work for [6,7].25.0

### Additional Notes

Nope.

### Describe your test plan

Standard QA procedure.

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -22,6 +22,24 @@
...
+        "OMNIBUS_SOFTWARE_VERSION": "7.25.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.25.0-rc.1",
+        "JMXFETCH_VERSION": "0.40.3",
+        "JMXFETCH_HASH": "6e8984c4b65cc553ab22b907b7947e00cd007c7aad7e85ef1799a1b532d52bee",
+        "MACOS_BUILD_VERSION": "6.25.0-rc.1",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.5"
+    },
+    "7.25.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.25.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.245.0-rc.1",
```
Comment (KSerrania) [inline release.json:36]:
```suggestion
        "OMNIBUS_SOFTWARE_VERSION": "7.25.0-rc.1",
```

**Review by olivielpeau — APPROVED** [review]
LGTM, pending confirmation that `SECURITY_AGENT_POLICIES_VERSION` doesn't need to be updated

File: release.json [hunk 2]
```diff
@@ -14,14 +14,32 @@
...
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.25.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.25.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.25.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.25.0-rc.1",
+        "JMXFETCH_VERSION": "0.41.0",
+        "JMXFETCH_HASH": "a0b545bdd38308d4d55b49e2c986c4d32fe1388d9bb265f16c1ea4fc545731e3",
+        "MACOS_BUILD_VERSION": "6.25.0-rc.1",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.5"
```
Comment (olivielpeau) [inline release.json:32]:
```suggestion
        "SECURITY_AGENT_POLICIES_VERSION": "v0.6"
```

File: release.json [hunk 3]
```diff
@@ -14,14 +14,32 @@
...
+    },
+    "7.25.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.25.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.25.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.25.0-rc.1",
+        "JMXFETCH_VERSION": "0.41.0",
+        "JMXFETCH_HASH": "a0b545bdd38308d4d55b49e2c986c4d32fe1388d9bb265f16c1ea4fc545731e3",
+        "MACOS_BUILD_VERSION": "7.25.0-rc.1",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.5"
```
Comment (olivielpeau) [inline release.json:41]:
```suggestion
        "SECURITY_AGENT_POLICIES_VERSION": "v0.6"
```

**Review by olivielpeau — APPROVED** [review]

**Pull request merged**